packages/trpc/server/routers/viewer/teams/inviteMember.handler.ts

```diff
@@ -135,7 +135,6 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
         inviteTeamOptions.isCalcomMember = false;
       }
 
-
```

Comment:
Unnecessary diff here.

Reply:
I think this got introduced when i ran `yarn lint -- --fix`